### PR TITLE
Issue #1520: Internship Thoughts Post

### PR DESCRIPTION
**Pull request by lrowang**
Hi Team!

PR for my blog post. Any and all feedback is welcome!
I left the tag category blank for the moment as I wasn't sure how we wanted to class this. I'm open to things like `learning` or `development` (though that term seems to have a lot of definitions in this world of ours).

Ro

### CODE REVIEW

**Comment by AnneTee** [comment]
@lrowang I just noticed this PR when I was doing other work on the site - please let me know next week if you still need a review, I'm happy to take a look!

**Comment by lrowang** [comment]
Hello Team!

I updated my blog post per @chrisarusso 's direction so that it's more focused on the internship experience! A couple things:
- How do I center the inline images? Mine are left justified, but I looked at some of the other blog posts and they seem to just be centered by default so I'm a bit stuck there.
- What's the best way to do endnotes/footnotes in a blog post (see `The Learning` section)?

All feedback welcome!

File: _posts/2016-08-02-internship-thoughts.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "A Savas Labs Internship"
+date: 2016-08-02
```
Comment (kostajh) [inline _posts/2016-08-02-internship-thoughts.md:4]: We'll want to update this & also rename the file when it's ready to publish :)

**Comment by kostajh** [comment]
@lrowang this post has footnotes if you'd like to see how it's implemented there: `/2016/04/20/squabble-comments.html`

File: _posts/2016-08-02-internship-thoughts.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+If you've ever contemplated a drastic career change, then you've probably thought about what it'd be like to start at the beginning again at some point.
+
+In this post, I'd like to share my 3 month internship experience for anyone who's considering making the switch to Web Development, or who may be curious about the learning environment here at Savas.
+
+## Visiting the Team
+
+Since I’m located [2,845 miles](https://www.google.com/maps/dir/Portland,+OR/Durham,+NC/@39.4988195,-118.778336,4z/data=!4m13!4m12!1m5!1m1!1s0x54950b0b7da97427:0x1c36b9e6f6d18591!2m2!1d-122.6764816!2d45.5230622!1m5!1m1!1s0x89ace471120f66f1:0xe17ada898a46f27e!2m2!1d-78.898619!2d35.9940329) from Savas Labs, one of my biggest concerns starting out as a newbie was the added challenge of telecommuting. To get me up and running as fast as possible, [Chris](http://savaslabs.com/team/chris-russo/) and I decided I’d fly to Durham to meet the team and work in the office together for my first week.
```
Comment (kostajh) [inline _posts/2016-08-02-internship-thoughts.md]: Since the time your internship started,  we became majority non Durhamites so maybe rephrase this as Savas Labs HQ?

File: _posts/2016-08-02-internship-thoughts.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## The Learning
+
+**There is a. lot. to. learn** (I mean, it can’t _all_ be roses right?).
+
+Though comfortable with programming syntax when I started (a significant portion of my last job was writing custom object-oriented programs for mechanical systems), I had never heard of git before or had any idea how to use the command line. I had, essentially, _zero_ Web Development experience*.
+
+To be honest, I wasn’t expecting the sheer breadth and volume of material I’d need to know just to reach a modicum level of proficiency. I spent the first month figuring out the foundations -- making pull requests, figuring out how to download software from the command line (don’t even get me started on virtualization)-- before I could even start to dip my toes into Drupal development. This sounds naive now, but I was genuinely surprised when I realized that Web Development was going to be much more [difficult](https://web.archive.org/web/20160507042856/http://www.codem0nk3y.com/wp-content/uploads/2012/04/cms-learning-curve.png) than reading a few good books on Drupal and CSS!
+
+<img src="/assets/img/blog/internship-learning-cartoon.png" alt="So much to learn!">
```
Comment (kostajh) [inline _posts/2016-08-02-internship-thoughts.md]: ha, love it!

**Comment by AnneTee** [comment]
@lrowang You can add classes to your blog images to center them and give them a little shadow - you'll probably want `class="blog-image-xl"` for all/most of them, which makes the image centered and sets its max width to the same as the max width of the text. See the styles [here and below](https://github.com/savaslabs/savaslabs.github.io/blob/master/assets/styles/scss/components/_post.scss#L80) for more info.

The post is awesome!! Excellent images, good tone and a nice overview of your experience.

**Comment by lrowang** [comment]
Thank you for the help and feedback @AnneTee and @kostajh !

I tweaked the images and added some tags but left the date as is for now until we decide when it'll be published. If anyone else has time to give it another 👀  that'd be great!

**Comment by AnneTee** [comment]
@lrowang looks great! 👍 ➕ :sparkles:

**Comment by chrisarusso** [comment]
@lrowang there are some failing tests, please ask @AnneTee as to how to fix those. Reading now though, and it's great!

**Comment by yonas** [comment]
@lrowang Great work! It was very fun to read and the articles you linked to are gold.

I just have a few minor grammar suggestions here:

https://gist.github.com/yonas/e04759f8affcfeeb36b7073d7c3610d3/revisions?diff=split

**Comment by AnneTee** [comment]
@lrowang You'll also need to close this PR and open a new one against the source branch instead of master - see the #savaslabs-com channel for me, and please ping me with any questions!
